odl/tomo/backends/astra_setup.py

```diff
@@ -339,7 +347,7 @@ def astra_projection_geometry(geometry):
             'parallel3d', det_width, det_height, det_row_count,
             det_col_count, angles)
 
-    elif isinstance(geometry, CircularConeFlatGeometry):
+    elif False and isinstance(geometry, CircularConeFlatGeometry):
```

Comment:
Imo, remove this whole elif clause unless ASTRA optimizes for the `cone` geometry.